alea/test/gtest_util.hpp

```diff
@@ -0,0 +1,60 @@
+#pragma once
```

Comment:
Shouldn't this be promoted to a higher level in ALPSCore's directory hierarchy?

Reply:
Probably, @galexv what do you think?